cryptographic_estimators/SDFqEstimator/SDFqAlgorithms/leebrickell.py

```diff
@@ -26,27 +26,29 @@
 class LeeBrickell(SDFqAlgorithm):
     def __init__(self, problem: SDFqProblem, **kwargs):
         """
-        Construct an instance of Lee-Brickells's estimator [LB88]_
-        expected weight distribution::
-            +--------------------------------+-------------------------------+
-            | <----------+ n - k +---------> | <----------+ k +------------> |
-            |               w-p              |              p                |
-            +--------------------------------+-------------------------------+
-        INPUT:
-        - ``problem`` -- SDProblem object including all necessary parameters
+        Construct an instance of Lee-Brickell's estimator [LB88].
```

Comment:
Missing `_`. But, as you said, this can be fixxed later. Just dont forget it

Reply:
Fixed :+1: